File: customviewjava/src/main/java/com/codekul/customviewjava/MainActivity.java
package com.codekul.customviewjava;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        CustomET customET = new CustomET(this);
        setContentView(customET);
    }
}

File: customviewkotlin/src/main/java/com/codekul/customviewkotlin/MainActivity.kt
package com.codekul.customviewkotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(MyView(context = this@MainActivity, attrs = null))
        // if you want to use myview in xml search for View in pallet :P
       // R.layout.activity_main
    }
}
